rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -23,6 +23,10 @@ import cdm.product.collateral.*
 
 import cdm.mapping.config.*
 
+type TransferableProduct:  <"A TransferableProduct is a type of Product which can be used in a SettlementCommitment for a basic cash settled trade of either an Asset with or without the addition of specific EconomicTerms.">
+    asset Asset (1..1)
+    economicTerms EconomicTerms (0..1)
```

Comment:
As discussed live: potentially revisit this type and its use in `SettlementCommitment`, but ok as is for Phase 1.

Reply:
Will be updated in later phases